src/transaction.ts

```diff
@@ -76,6 +78,7 @@ export interface ExecuteSqlRequest extends Statement, RequestOptions {
   partitionToken?: Uint8Array | string;
   seqno?: number;
   queryOptions?: IQueryOptions;
+  requestOptions?: IRequestOptions;
```

Comment:
Probably should not use the same property name as in `CommitOptions` interface since `CommitOptions.requestOptions` will only support `priority` parameter and in both `ExecuteSqlRequest` and `BatchUpdateOptions` same property name will support `requestTag` parameter